service.sh

```diff
@@ -27,11 +29,13 @@ beertime() {
     fi
   }
 
-  local hrs=$(date +"%H")
-  local day=$(date +"%w")
+  local hrs=
+  hrs=$(date +"%H")
+  local day=
+  day=$(date +"%w")
   case $day in
-    [1-4]*) beer_o_clock 17 $hrs;; # mon - thurs
-    5) beer_o_clock 16 $hrs;; # fri
+    [1-4]*) beer_o_clock 17 "${hrs}";; # mon - thurs
```

Comment:
I prefer `${hrs}` for a couple of  reasons: 1) BASH's String functions are immediately available, e.g. `first_digit=${hrs:0:1}` and `hours=${hrs-${default_hrs}}` and 2) it's easier to distinguish variables from strings or other variables when there's a lot going on in the string, e.g. `It took ${hrs}.${minutes}.${seconds}` vs `It took $hrs.$minutes.$seconds`.

There have been cases where BASH gets confused about where the variable ends because I've used `$hrs` but I can't think of any good example now. It's safe to say it's rare, `$hrs` normally works just fine.